example/package.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "mobx-react-lite-example",
+    "version": "0.0.1",
+    "main": "index.js",
```

Comment:
I don't think there should be a `main` here as it's not a module to be consumed.

Reply:
yeah, I'll trim that out, just came with the "init" command from yarn.